assets/src/edit-story/components/carousel/zoomSelector/zoomSelector.js

```diff
@@ -18,20 +18,36 @@
  * External dependencies
  */
 import { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
 import { __, _x } from '@web-stories-wp/i18n';
 /**
  * Internal dependencies
  */
 import { ZOOM_SETTING } from '../../../constants';
 import { useLayout } from '../../../app/layout';
-import { DropDown } from '../../form';
-import { Placement } from '../../popup';
+import { DropDown, PLACEMENT } from '../../../../design-system';
+
+const StyledDropDown = styled(DropDown)`
+  button {
+    padding: 8px;
+
+    span {
+      padding: 0;
+    }
+  }
+`;
```

Comment:
Sizing and padding aren't correct [according to Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=2241%3A2):

![Screen Shot 2021-03-30 at 09 15 02](https://user-images.githubusercontent.com/637548/112994934-bd984080-9138-11eb-8d69-1a5a73f3d3b9.png)

Note that the zoom dropdown should align with the help center in height and vertical alignment.

Also, there should *not* be a dropdown arrow in this one(?), but perhaps double-check with UX on that one.

Reply:
> Sizing and padding aren't correct [according to Figma]
I'll fix that right up

> Note that the zoom dropdown should align with the help center in height and vertical alignment.

Ah, the alignment is the same but not the height. I'll fix that too.

> Also, there should _not_ be a dropdown arrow in this one(?), but perhaps double-check with UX on that one.

Already chatted with them - they said they wanted the arrow: https://xwp.slack.com/archives/C01ESBB8ZN1/p1617050704003700

Reply:
> > Sizing and padding aren't correct [according to Figma]
> 
> I'll fix that right up
> 
> > Note that the zoom dropdown should align with the help center in height and vertical alignment.
> 
> The height shouldn't be the same according to figma, no?

There's inconsistencies in the designs. Have a thread going here:
https://xwp.slack.com/archives/C01ESBB8ZN1/p1617113850007600

Reply:
> The height shouldn't be the same according to figma, no?

`36 === 36`, no?

Reply:
Oh I had my dimensions flipped. Long day 🤕

Reply:
Looks good now - excellent job!